### DISCUSSION TITLE
enableing Modbus Power Control (remotecontrol power) not working ... how to?

### DISCUSSION THREAD

**Discussion by dvoelker** [wills106/homeassistant-solax-modbus#583] [Q&A]
Hey, I'm trying to enable the Modbus Power Control (remotecontrol power). I belive I have to klick on unlock inverter.

![image](https://github.com/wills106/homeassistant-solax-modbus/assets/28888107/0fe7593a-89e0-461e-98de-6c106e5ae2b7)

And the modbus Power Control stays disabled

![IMG_20231028_212951](https://github.com/wills106/homeassistant-solax-modbus/assets/28888107/c80b4e04-9169-4a78-9aa4-7adf2a45e526)

In the Logfile I find the following:
![image](https://github.com/wills106/homeassistant-solax-modbus/assets/28888107/93a741c7-0f67-4534-a3e9-5dbc47c4b160)

But I my case the nice power control dashboard does not start. 
I have a solax X3 G4 8kw
HA Version:
Home Assistant 2023.10.5
Supervisor 2023.10.1
Operating System 11.1
Frontend 20231005.0 - latest

Is there anyone out there that could give a hint?

Thanks a lot :)

**Comment by infradom** [comment]
There is not such thing as a 'power control dashboard', so it is unclear what you mean.
You can create a page with a card like this: 
<img width="482" alt="image" src="https://github.com/wills106/homeassistant-solax-modbus/assets/11804014/efb720fe-1919-48c2-b802-fe1119d8ee9b">

It will help you understand the mechanisms behind modbus power control.

Once you have created this, you can press the trigger button(link) to activate your configuration for the configured mode and duration.

In real life, you will use it in combination with an automation.

See wiki page https://github.com/wills106/homeassistant-solax-modbus/wiki/Modbus-Power-Control-(remotecontrol-power)
